server/app/controllers/DisplayableMessage.java

```diff
@@ -0,0 +1,41 @@
+package controllers;
+
+import views.components.ToastMessage.ToastType;
+
+/** Model for user feedback messages that appear on the screen with a certain severity. */
```

Comment:
suggestion (blocking): "model" in the context of this code base refers to EBean models. I get that this is meant to be a data class representing a domain object, i.e. a model, but would prefer to avoid overloading the term in the code itself (including comments and docs). Maybe use "data class" instead of model?